.github/workflows/ci-components.yml

```
@@ -48,7 +48,8 @@ jobs:
         try-scenario:
          # - ember-lts-3.24
           - ember-lts-3.28
-          - ember-release
+          - ember-lts-4.8
+         # - ember-release
```

```suggestion
          - ember-release
```

This should def stay in, OR we should add couple more of the 4.X releases.

---

> This should def stay in

Apologies if the language in [RFC DS-051](https://docs.google.com/document/d/1u1-lTpiLN791DtXCZsNdFayt8NaDsaFSultBcec6TAA/edit#bookmark=id.dzbnfq85y6gq) was not clear. The proposition to remove `ember-release` (currently 4.12.1) is because it is already covered by our default tests (outside the `ember-try` suite), and we're committing to keeping up with it. I tried to highlight this in the PR description too. What do you think would be the value of running our tests with 4.12.1 twice?

> OR we should add couple more of the 4.X releases

The only other 4.x LTS is 4.4, and considering Ember's commitment to backward compatibility, I'm not sure of the value of testing with both 4.4 and 4.8.

I think it's important to keep our matrix down to a reasonable number, as each entry has a considerable impact on the test duration for each PR.